bin/lib/create.js

```diff
@@ -259,41 +273,53 @@ exports.create = function(project_path, config, options, events) {
 
         setShellFatal(true, function() {
             var project_template_dir = options.customTemplate || path.join(ROOT, 'bin', 'templates', 'project');
+            var app_path = path.join(project_path, 'app', 'src', 'main');
```

Comment:
The plan is to support managing the old and existing structures while only being able to create new structures for the time being.  Of course, there's a debate on how update works, and we may just toss the structure altogether.  Who knows?